### PR TITLE
Intel/CI: Fix missing test names in summary

### PR DESCRIPTION
**Pull request by Juee14Desai**
OneCCL and OneCCL GPU stages were missing test names and correct
test numbers

SHMEM is missing PRK and ISx test names from the summary.
This patch adds the capability to find the test names and add
them to the passed and failed test lists for verbose reporting.
Tests will have the type of SHMEM test added as a prefix to them.

### CODE REVIEW

File: contrib/intel/jenkins/summary.py [hunk 1]
```diff
@@ -501,15 +509,21 @@ def check_prk(self, line, log_file=None):
                     f"{token}"
                 )
 
+        self.previouse = line
```
Comment (zachdworkin) [inline contrib/intel/jenkins/summary.py:512]: This has a typo previouse -> previous

**Review by zachdworkin — CHANGES_REQUESTED** [review]
Can you fix my typo and fix the missing letters in the commit message for shmem? The U is missing from UH the P from PRK and the I from ISx

File: contrib/intel/jenkins/summary.py (hunk 1 above)
Comment (Juee14Desai) [reply contrib/intel/jenkins/summary.py]: Done.

**Review by zachdworkin — APPROVED** [review]
I think the summary stage looks good. Can you also take a look at the summary stage before merging please?

**Comment by j-xiong** [comment]
Please fix the indentation in the commit message for the second commit (shmem summary).

**Comment by Juee14Desai** [comment]
Indentation has been fixed in recent commit.

**Pull request merged**